test_reproducibility.sh

```diff
@@ -21,7 +21,9 @@ md5_util() {
 }
 
 non_deploy_jar_md5_sum() {
-    find bazel-bin/test -name "*.jar" ! -name "*_deploy.jar" ! -path 'bazel-bin/test/jmh/*' | xargs -n 1 -P 5 $(md5_util) | sort
+    # Exclude *-docs.jar (javadoc jars from maven_export in rules_jvm_external)
+    # because javadoc generates non-deterministic output.
```

Comment:
Do you maybe remember what was the issue here? Maybe we should fix it upstream in the scaladoc tool or add some options to make the outputs more deterministic